bot/exts/moderation/infraction/_scheduler.py

```diff
@@ -48,11 +48,32 @@ async def reschedule_infractions(self, supported_infractions: t.Container[str])
 
         infractions = await self.bot.api_client.get(
             'bot/infractions',
-            params={'active': 'true'}
+            params={
+                'active': 'true',
+                'ordering': 'expires_at',
+                'permanent': 'false',
+                'types': ','.join(supported_infractions),
+            },
```

Comment:
I know this was using single quotes already, but [our style guide](https://pythondiscord.com/pages/guides/pydis-guides/contributing/style-guide/#quote-marks) says that we should prefer single quotes over double quotes.
```suggestion
            params={
                "active": "true",
                "ordering": "expires_at",
                "permanent": "false",
                "types": ",".join(supported_infractions),
            },
```